static/usage/v6/modal/can-dismiss/prevent-swipe-to-close/angular/example_component_html.md

```diff
@@ -0,0 +1,30 @@
+```html
+<div class="ion-page" #page>
+  <ion-header>
+    <ion-toolbar>
+      <ion-title>App</ion-title>
+    </ion-toolbar>
+  </ion-header>
+  <ion-content class="ion-padding">
+    <ion-button id="open-modal" expand="block">Open</ion-button>
+    <ion-modal #modal trigger="open-modal" [canDismiss]="canDismiss" [presentingElement]="page">
```

Comment:
In React we call the presenting el var `presentingElement` but in Angular and Vue we call it `page`. Can we make the vars consistent? I believe in other demos we use the `presentingElement` naming.